seqr/views/apis/variant_search_api_tests.py

```diff
@@ -160,7 +160,7 @@ def test_query_variants(self, mock_get_variants, mock_get_gene_counts):
             export_content[1],
             ['21', '3343400', 'GAGA', 'G', 'WASH7P', 'missense_variant', '', '', '', '', '', '', '', '', '', '', '', '',
              '', 'ENST00000623083.3:c.1075G>A', 'ENSP00000485442.1:p.Gly359Ser', '', '', '', '1',
-             'Tier 1 - Novel gene and phenotype (None)|Review (None)', '', '2', '', '', 'NA19675:1:46.0:0.702127659574', 'NA19679:0:99.0:0.0'])
```

Comment:
This should not have changed, its an important thing to test. You need to change how you modified the fixture data so this is still correctly getting a tagged variant

Reply:
Rolled back.